packages/php/woorelease-extension/src/Utils/Nvm.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * NVM class.
+ *
+ * @package Automattic\WooCommerce\Grow\WR\Utils
+ */
+
+namespace Automattic\WooCommerce\Grow\WR\Utils;
+
+use Exception;
+use WR\Tools\Utils;
+
+/**
+ * Class NVM
+ *
+ * Check if `nvm` CLI tool is installed.
+ */
+class Nvm {
+
+	/**
+	 * Checks if `nvm` is installed.
+	 *
+	 * @return bool True if `nvm` is installed, false otherwise.
+	 */
+	public static function is_nvm_exists() {
```

Comment:
Changing the name helps this to read better.

```suggestion
	public static function does_nvm_exist() {
```